app/common/src/cljc/teet/authorization/authorization_check.cljc

```diff
@@ -46,6 +46,7 @@
                  (and (if (seq projects)
                         ;; Project specific permission: check project id is included
                         (and project-id
+                             ;; FIXme project-id is not db/id
```

Comment:
will this be fixed in another issue?

Reply:
Sorry, forgot to remove the comment. I already fixed this separately to master, see `teet.db-api.core`:
```
(defn project-id->db-id [db project-id]
  (if (vector? project-id)
    (:db/id (d/pull db '[:db/id :thk.project/id]
                    project-id))
    project-id))
```